src/common/hooks/useOrder/useOrder.js

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { ORDERS_API } from '@folio/stripes-acq-components';
+
+export const useOrder = (orderId) => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'purchase-order' });
+
+  const {
+    data,
+    isLoading,
+  } = useQuery({
+    queryKey: [namespace, orderId],
+    queryFn: async () => ky.get(`${ORDERS_API}/${orderId}`).json(),
```

Comment:
you need permissions to load

Reply:
Good point. Moreover, this requires a new okapi interface (`orders`).